From 450729be2252634893d7ea266aeb30a5469233f0 Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Mon, 23 Jun 2014 21:41:52 -0400
Subject: [scsi] lpfc: Using plain integer as NULL pointer

Message-id: <1403559726-20488-11-git-send-email-revers@redhat.com>
Patchwork-id: 84434
O-Subject: [RHEL7.1 e-stor PATCH 10/24] scsi: lpfc: Using plain integer as NULL pointer
Bugzilla: 1088574
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dick Kennedy <rkennedy@Emulex.Com>

Submitted by Daeseok Youn Daeseok.Youn@emulex.comSubmitted by Daeseok Youn Daeseok.Youn@emulex.com[SCSI

From 9e7478f6e953fac5b2bef0f5abe76fe8dc9e59d1 Mon Sep 17 00:00:00 2001
From: Daeseok Youn daeseok.youn@gmail.com
Date: Fri, 21 Feb 2014 09:03:32 +0900
Subject: [PATCH] [SCSI] lpfc 8.3.43: use NULL instead of 0 for pointer

sparse says:
drivers/scsi/lpfc/lpfc_sli.c:16547:37: warning:
 Using plain integer as NULL pointer

Signed-off-by: Daeseok Youn daeseok.youn@gmail.com

http://marc.info/?l=linux-scsi&m=139294505704556&w=2
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/lpfc/lpfc_sli.c b/drivers/scsi/lpfc/lpfc_sli.c
index db2100d..6bb51f8 100644
--- a/drivers/scsi/lpfc/lpfc_sli.c
+++ b/drivers/scsi/lpfc/lpfc_sli.c
@@ -16773,7 +16773,7 @@ lpfc_drain_txq(struct lpfc_hba *phba)
 {
  LIST_HEAD(completions);
  struct lpfc_sli_ring *pring = &phba->sli.ring[LPFC_ELS_RING];
- struct lpfc_iocbq *piocbq = 0;
+ struct lpfc_iocbq *piocbq = NULL;
  unsigned long iflags = 0;
  char *fail_msg = NULL;
  struct lpfc_sglq *sglq;
-- 
1.7.1